src/libstd/net/ip.rs

```diff
@@ -722,25 +732,26 @@ mod tests {
             assert_eq!(ip.is_multicast(), multicast);
             assert_eq!(ip.is_broadcast(), broadcast);
             assert_eq!(ip.is_documentation(), documentation);
+            assert_eq!(ip.is_this_network(), this_network);
         }
 
-        //    address                unspec loopbk privt  linloc global multicast brdcast doc
-        check(&[0, 0, 0, 0],         true,  false, false, false, true,  false,    false,  false);
-        check(&[0, 0, 0, 1],         false, false, false, false, true,  false,    false,  false);
-        check(&[1, 0, 0, 0],         false, false, false, false, true,  false,    false,  false);
-        check(&[10, 9, 8, 7],        false, false, true,  false, false, false,    false,  false);
-        check(&[127, 1, 2, 3],       false, true,  false, false, false, false,    false,  false);
-        check(&[172, 31, 254, 253],  false, false, true,  false, false, false,    false,  false);
-        check(&[169, 254, 253, 242], false, false, false, true,  false, false,    false,  false);
-        check(&[192, 0, 2, 183],     false, false, false, false, false, false,    false,  true);
-        check(&[192, 1, 2, 183],     false, false, false, false, true,  false,    false,  false);
-        check(&[192, 168, 254, 253], false, false, true,  false, false, false,    false,  false);
-        check(&[198, 51, 100, 0],    false, false, false, false, false, false,    false,  true);
-        check(&[203, 0, 113, 0],     false, false, false, false, false, false,    false,  true);
-        check(&[203, 2, 113, 0],     false, false, false, false, true,  false,    false,  false);
-        check(&[224, 0, 0, 0],       false, false, false, false, true,  true,     false,  false);
-        check(&[239, 255, 255, 255], false, false, false, false, true,  true,     false,  false);
-        check(&[255, 255, 255, 255], false, false, false, false, false, false,    true,   false);
+        //    address                unspec loopbk privt  linloc global multicast brdcast doc    this_net
+        check(&[0, 0, 0, 0],         true,  false, false, false, false,  false,    false,  false, true);
```

Comment:
Adding the space before the `multicast` column misaligns it and all of the following.
Wouldn't it be possible to remove a space before the `false` in the first two rows and avoid changing the whitespace in the remaining part of the table?